fusillade/resource.py

```diff
@@ -140,6 +141,39 @@ def actions(self):
             self._actions = self._actions.split(' ')
         return self._actions
 
+    def add_actions(self, actions: List[str]):
+        _actions = set(actions)
+        _actions.update(self.actions)
+        self.cd.update_object_attribute(self.object_ref,
+                                        [UpdateObjectParams(
+                                            self._facet,
+                                            'actions',
+                                            ValueTypes.StringValue,
+                                            ' '.join(_actions),
+                                            UpdateActions.CREATE_OR_UPDATE,
+                                        )])
+        self._actions = None
+
+    def remove_actions(self, actions: List[str]):
+        _actions = set(self.actions) - set(actions)
+        self.cd.update_object_attribute(self.object_ref,
+                                        [UpdateObjectParams(
+                                            self._facet,
+                                            'actions',
+                                            ValueTypes.StringValue,
+                                            ' '.join(_actions),
+                                            UpdateActions.CREATE_OR_UPDATE,
+                                        )])
+        # TODO this actions from all policies
```

Comment:
The language in this TODO is difficult to understand.

Reply:
will fix